src/style.css

```diff
@@ -0,0 +1,125 @@
+
+.todo-container {
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  max-width: 400px;
+  border-radius: 3px;
+  justify-content: center;
+  margin-top: 50px;
+  text-align: left;
+
+  /* border: 1px solid black; */
+  box-shadow: rgba(20, 18, 18, 0.2) 0 2px 8px 0;
+}
+
+.todo-header > h1 {
+  font-size: 18px;
+  font-weight: 500;
+  margin: 0;
+  color: #525252;
+}
+
+#refresh {
+  margin-right: 0;
+}
+
+.todo-header #refresh {
+  color: grey;
+}
+
+.main-container {
+  text-align: left;
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  max-width: 400px;
+  border-radius: 3px;
+  justify-content: center;
+  margin-top: 150px;
+  height: 60vh;
+
+  /* border: 1px solid black; */
+}
+
+.todo-header {
+  margin-top: -200px;
+  padding: 10px;
+  border-bottom: 1px solid #ebebeb;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 40px;
+}
+
+.form-container {
+  padding: 10px;
+}
+
+.todo-list {
+  margin-left: -40px;
+}
+
+.todo-list > li {
+  font-size: 14px;
+  font-weight: 300;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  align-content: center;
+}
+
+.add-new-task-container {
+  display: flex;
+  flex-direction: row;
+  gap: 170px;
+}
```

Comment:
- [ ] Please make sure that the add button doesn't stick out of the design to match the original template design.

![image](https://user-images.githubusercontent.com/62290545/214043292-8c1173fa-f13a-4169-a387-0a69c67f8849.png)

Reply:
@ShadyShawkat  thank you, I corrected it.
![Screenshot from 2023-01-23 17-34-35](https://user-images.githubusercontent.com/55842414/214066031-c11d14f4-adc6-46f2-a75a-5ca62d33342e.png)